edx_proctoring/constants.py

```diff
@@ -68,7 +68,11 @@
 
 PING_FAILURE_PASSTHROUGH_TEMPLATE = 'edx_proctoring.{}_ping_failure_passthrough'
 
-ONBOARDING_PROFILE_API = 'edx_proctoring.onboarding_profile_api'
+ONBOARDING_PROFILE_API = (
+    settings.PROCTORING_SETTINGS['USE_ONBOARDING_PROFILE_API'] if
+    'USE_ONBOARDING_PROFILE_API' in settings.PROCTORING_SETTINGS
+    else getattr(settings, 'USE_ONBOARDING_PROFILE_API', False)
```

Comment:
Why do we have to check settings.PROCTORING_SETTINGS and settings?

Reply:
I'm just aiming to be consistent with the rest of the settings in this file (I'm assuming it's for open edX considerations) but I guess it technically isn't necessary

Reply:
Got it that makes sense!